main.go

```diff
@@ -128,6 +137,23 @@ func ParseEnvVars(envVars []string) ([]config.HostEnvVar, error) {
 	return vars, nil
 }
 
+func ParseFiles(files []string) ([]config.FileInjection, error) {
+	FileInjections := []config.FileInjection{}
```

Comment:
nitpick: Using capital leters for local variables is a bit weird. :)

Reply:
argh, agreed, autocomplete made me do this and I didn't see it 😄